NOTICE.TXT

```diff
@@ -0,0 +1,5 @@
+Elasticsearch
+Copyright 2012-2016 Elasticsearch
+
+This product includes software developed by The Apache Software
```

Comment:
Is this true? This plugin isn't developed by Apache and I don't think it includes any Apache works, does it?